### PR TITLE
magento/devdocs#5361:  404 external links

### PR DESCRIPTION
**Pull request by atwixfirster**
## Purpose of this pull request

This pull request (PR) fixes https://github.com/magento/devdocs/issues/5361.

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: guides/v2.2/javascript-dev-guide/widgets/widget-row-builder.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ The RowBuilder [widget](https://glossary.magento.com/widget/) allows you to buil
 
 The RowBuilder widget can be used only on the frontend area.
 
-The RowBuilder widget source is [lib/web/mage/row-builder.js][].
+The RowBuilder widget source is [lib/web/mage/row-builder.js]({{ site.mage2bloburl }}/{{ page.guide_version }}/app/code/Magento/Theme/view/frontend/web/js/row-builder.js).
```
Comment (dshevtsov) [inline guides/v2.2/javascript-dev-guide/widgets/widget-row-builder.md]: Revert this change and fix the link in _Link Definitions_ (at the end of file).

Comment (atwixfirster) [reply guides/v2.2/javascript-dev-guide/widgets/widget-row-builder.md]: yep, good point

**fixed**

**Comment by dshevtsov** [comment]
running tests

File: guides/v2.2/javascript-dev-guide/widgets/widget-row-builder.md [hunk 2]
```diff
@@ -260,5 +260,5 @@ As a result, we see the RowBuilder widget example with 5 rows as max.
 ![RowBuilder widget with warning message Example]({{ site.baseurl }}/common/images/widget/row-builder-widget-warning-message-result.png)
 
 <!-- Link Definitions -->
-[lib/web/mage/row-builder.js]: {{ site.mage2bloburl }}/{{ page.guide_version }}/lib/web/mage/row-builder.js
+[lib/web/mage/row-builder.js]: ({{ site.mage2bloburl }}/{{ page.guide_version }}/app/code/Magento/Theme/view/frontend/web/js/row-builder.js)
```
Comment (dshevtsov) [inline guides/v2.2/javascript-dev-guide/widgets/widget-row-builder.md]:
```suggestion
[lib/web/mage/row-builder.js]: {{ site.mage2bloburl }}/{{ page.guide_version }}/app/code/Magento/Theme/view/frontend/web/js/row-builder.js
```

Comment (atwixfirster) [reply guides/v2.2/javascript-dev-guide/widgets/widget-row-builder.md]: yes......

**fixed**

**Comment by dshevtsov** [comment]
running tests
